lib/Imagine/Gmagick/Image.php

```diff
@@ -283,6 +283,14 @@ private function applyImageOptions(\Gmagick $image, array $options, $path)
             $image->setCompressionQuality($options['jpeg_quality']);
         }
 
+        if (isset($options['webp_quality']) && in_array($format, array('webp'))) {
+            $image->setCompressionQuality($options['webp_quality']);
+        }
+
+        if (isset($options['webp_lossless']) && in_array($format, array('webp'))) {
+            throw new NotSupportedException('Gmagick does not support webp_lossless option');
```

Comment:
I’m not sure if `NotSupportedException`s should get thrown for unsupported options. I created the pull request #591 for this.